### DISCUSSION TITLE
fooocus on docker desktop - where is the run.bat file?

### DISCUSSION THREAD

**Discussion by I4m5umm3r** [lllyasviel/Fooocus#3687] [Q&A] [answered]
Hi
I'm wanting to add "enable describe uov image"
can one edit the run.bat from within docker? If so, where the heck is it?
am I on the right track?
cheers in advance.

**Comment by mashb1t** [comment, marked as answer]
you can add arguments to the env var CMDARGS in your docker-compose.yml
